### PR TITLE
[18.0][MIG] partner_company_group: Migration to 18.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**
Depend on:
- [x] https://github.com/OCA/partner-contact/pull/1871
- [x] https://github.com/OCA/partner-contact/pull/1872
- [x] https://github.com/OCA/partner-contact/pull/1873
- [x] https://github.com/OCA/partner-contact/pull/1874

oca-port: [blacklist PR(s) , 1243, 1257, 1657 for partner_company_group](https://github.com/OCA/partner-contact/pull/1870/commits/eca7ab3ac5561db528903e4b2d47f729183ae121)

### CODE REVIEW

File: sale_partner_company_group/tests/test_sale_partner_company_group.py [hunk 1]
```diff
@@ -10,16 +10,6 @@ class TestSalePartnerCompanyGroup(TestBasePartnerCompanyGroup):
...
-        # Remove this variable in v16 and put instead:
-        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
-        DISABLED_MAIL_CONTEXT = {
-            "tracking_disable": True,
-            "mail_create_nolog": True,
-            "mail_create_nosubscribe": True,
-            "mail_notrack": True,
-            "no_reset_password": True,
-        }
-        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
```
Comment (nguyenminhchien) [inline sale_partner_company_group/tests/test_sale_partner_company_group.py]: TestBasePartnerCompanyGroup inherits from BaseCommon which has already disabled mail context. So, i removed these lines.

**Comment by pedrobaeza** [comment]
Please only one module per PR.

**Comment by nguyenminhchien** [comment]
> Please only one module per PR.

Hi @pedrobaeza , updated!

**Review by mmequignon — APPROVED** [review]

**Review by BT-dmoreno — APPROVED** [review]
LGTM 👍🏻

**Review by sebalix — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Hello 👋🏻 

I fail to understand why this module was migrated.

I think this module is a left-over from https://github.com/OCA/partner-contact/pull/1347/commits/7f3f531e6dc55ea12ecd9f726ed6013ad46a3fcb, probably meant -at the time- to offer an easy migration path for people with the module already installed previous to the refactor/split.

But I don't think we should've migrated this 🤔

**Comment by ivantodorovich** [comment]
Following up on my previous comment:
- https://github.com/OCA/partner-contact/pull/2148

**Pull request merged**